tests/test_model_interface/test_keras_tensorflow_model.py

```diff
@@ -74,4 +74,6 @@ def test_model_output(self, sample_adultincome_query, public_data_object, predic
             predictval = self.sess.run(output_instance)[0][0]
         else:
             predictval = output_instance.numpy()[0][0]
-        pytest.approx(predictval, abs=1e-3) == prediction
+        assert predictval is not None
```

Comment:
Is this test failing always? Or only for some inputs?
It is okay for this PR, but I will look to fix this when I do the update for the pytorch/keras backend

Reply:
I think these tests always fail. We need a little bit more time and eyes to debug why these would fail.